### DISCUSSION TITLE
Connect Appium Server to a desktop browser possible?

### DISCUSSION THREAD

**Discussion by pandoras-toolbox** [appium/appium#18828] [Q&A]
Can desktop browsers be attached to the Appium Server? My motivation to do this is because of Flutter web apps where the widgets cannot be located with Selenium, a Flutter Appium Driver is required for this.

But when I tried to connect to a Chrome browser on my desktop with the capability (among others) browserName Chrome then Appium Server showed an error message that it cannot find the Gecko driver.

So I wonder if it is even possible to connect a desktop browser to Appium Server? If yes, then I made something wrong and can start trying again, provide you error messages, details and so on. If not, then I can give up trying this.

**Comment by KazuCocoa** [comment]
https://github.com/appium/appium-chromium-driver may help to do chrome browser automation via appium.
https://github.com/appium-userland/appium-flutter-driver itself doesn't support the usage, but probably can do something
